shared/src/ledger/vp_env.rs

```diff
@@ -27,6 +27,8 @@ pub enum RuntimeError {
     NumConversionError(TryFromIntError),
     #[error("Memory error: {0}")]
     MemoryError(Box<dyn std::error::Error + Sync + Send + 'static>),
+    #[error("IBC key error: {0}")]
+    IbcKeyError(String),
```

Comment:
It's not specific just to this PR, but I think maybe we should add an associated `type Error` to the `trait NativeVp`, so that each native VP impl can provide its own error type. Currently, we're just using this `Error` type imported from here and it's breaks the IBC module's encapsulation.

Reply:
Thank you for your suggestion! I've added `Error` for `NativeVp`.